src/cloud_provider/aws/kubernetes/helm_charts.rs

```diff
@@ -1254,85 +1239,90 @@ impl HelmChart for AwsVpcCniChart {
 
         match self.enable_cni_managed_by_helm(kubernetes_config, envs) {
             true => {
-                info!("Enabling AWS CNI support by Helm");
-
                 for kind in kinds {
-                    info!("setting annotations and labels on {}/aws-node", &kind);
-                    let steps = || -> Result<(), SimpleError> {
+                    // Setting annotations and labels on kind/aws-node
+                    let steps = || -> Result<(), CommandError> {
+                        let tag = format!("meta.helm.sh/release-name={}", self.chart_info.name);
```

Comment:
I would rename this to label as kubernetes naming

Reply:
Ok, done, thanks